app/components/AddWidgetModal.tsx

```diff
@@ -365,6 +383,7 @@ function AddWidgetModal({
                         <Grid.Column width={12}>
                             <WidgetListWrapper>
                                 <WidgetList divided>
+                                    {find(widgetDefinitions, { loaded: false }) && <LoadingOverlay />}
```

Comment:
Could we use here a native `Array.prototype.find` function?
(as suggested in our [UI Code Style guideline](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/435388492/UI+Code+Style#SHOULD%3A-Use-built-in-JS-functions-instead-of-utilities-functions-from-external-packages))

```suggestion
                                    {widgetDefinitions.find(widgetDefinition => !widgetDefinition.loaded) && (
                                        <LoadingOverlay />
                                    )}
```

Reply:
I recently introduced a bug by applying such change (`lodash` functions are null-safe as opposed to those from `Array` and  we are not 100% TS yet). So I'd prefer not to apply such changes blindly unless I'm 100% sure they are safe.

Reply:
Okay, seems lika a fair point 👌 
Thanks for the explanation 💪